.github/workflows/python.yml

```diff
@@ -116,6 +116,7 @@ jobs:
               run: |
                   source .env/bin/activate
                   pip install -r requirements.txt
+                  pip install -r dev_requirements.txt
```

Comment:
same question as above - can we remove the `                  pip install -r requirements.txt` line?